jobs/integr8ly/pds-testing-nightly-trigger.yaml

```diff
@@ -29,6 +29,7 @@
             CUSTOMER_ADMIN_PASSWORD=Password1
             CLEAN_RESOURCES=true
             MANIFEST_VERSION=master
+            INSTALL_BACKUPS=true
```

Comment:
Do we want to have this set to `true` by default?

Reply:
no preference. having it set to true will ease backup and restore testing. on the other hand might increase s3 usage

Reply:
I would go with `false` by default to be honest